packages/client/utils/taskStatus.ts

```diff
@@ -1,14 +1,14 @@
 import {TaskStatus, TaskStatusLabel} from '../types/constEnums'
 import {PALETTE} from '../styles/paletteV3'
 
-export const taskStatusLabels = {
+export const taskStatusLabels: Record<string, string> = {
```

Comment:
+1 `as const` (also makes this object readonly, so it'll throw an error if someone tries to mutate the object 🎉

Reply:
good point, done!